maxim/logger/together/client.py

```
@@ -0,0 +1,235 @@
+from collections.abc import Iterator, AsyncGenerator
+from uuid import uuid4
+from together.resources.chat import AsyncChatCompletions, ChatCompletions
+from together.types.chat_completions import ChatCompletionChunk
+from ..logger import Generation, Logger, Trace
+from .utils import TogetherUtils
+from ...scribe import scribe
+
+
+_INSTRUMENTED = False
+
+def instrument_together(logger: Logger) -> None:
```

Ruff (0.11.9)

`instrument_together` is too complex (36 > 10)

(C901)